plugins/base/src/main/kotlin/transformers/documentables/PropertiesMergerTransformer.kt

```diff
@@ -0,0 +1,118 @@
+package org.jetbrains.dokka.base.transformers.documentables
+
+import org.jetbrains.dokka.model.*
+import org.jetbrains.dokka.transformers.documentation.PreMergeDocumentableTransformer
+import org.jetbrains.kotlin.load.java.JvmAbi
+import org.jetbrains.kotlin.load.java.propertyNameByGetMethodName
+import org.jetbrains.kotlin.load.java.propertyNamesBySetMethodName
+import org.jetbrains.kotlin.name.Name
+
+/**
+ * This transformer is used to merge the backing fields and accessors (getters and setters)
+ * obtained from Java sources. This way, we could generate more coherent documentation,
```

Comment:
Is this transformer only applicable to java sources? If so:
1. what do you think about renaming it to `JavaPropertiesMergerTransformer`? 
2. Can we somehow filter processed items and avoid processing kotlin classes? Does it make sense? Although I don't know if we have the ability to extract that info from DClass. Or maybe you already have that filter somewhere, e.g via `KotlinVisibility`?

My main concern is that if we don't limit it, then kotlin classes might satisfy the same conditions as Java and it'll introduce bugs (i.e not display getters/setters of a kotlin class for explicit declarations). If it's only java classes that are processed (somehow), should be safer. But maybe this can never happen?

Reply:
Technically it can happen, I am not sure how I could limit it to only Java classes. Do you have any idea?